### PR TITLE
Replace cosign binary with bash wrapper [0.4]

### PR DESCRIPTION
**Pull request by simonbaird**
Same as #1873 but with an extra hack-cherry on top.

Ref: https://issues.redhat.com/browse/EC-817

### CODE REVIEW

**Comment by lcarva** [comment]
Ha! I love it. If we're doing that approach, let's revert the change to the Task. This will tell us for user if the wrapper is working as intended.

**Review by lcarva — APPROVED** [review]

**Comment by simonbaird** [comment]
I want to merge this a little after merging #2000 due to https://issues.redhat.com/browse/CLOUDWF-10542 .

**Pull request merged**
